models/public/netvlad/netvlad.md

```diff
@@ -0,0 +1,93 @@
+# netvlad
```

Comment:
Looks like this is a port to TensorFlow of the original MATLAB model, so it should be called `netvlad-tf`.

Reply:
Renamed